.gitignore

```diff
@@ -16,5 +12,5 @@ src/*.css
 dist/
 build/
 
-## Karma code coverage reports
+## Code coverage reports
```

Comment:
This file is wrong, see https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/.gitignore.temp

Reply:
Fixed.